packages/api/src/codegen/languages/typescript/index.ts

```diff
@@ -230,20 +230,19 @@ export default class TSGenerator extends CodeGenerator {
         let filePath = sourceFile.getFilePath().toString();
         filePath = filePath.substring(1);
 
+        /**
+         * It's not Prettier-level of nice but  `ts-morph` offers a method of using the TS
+         * formatter for formatting code which we can use to make our generated SDK not look like
+         * total garbage.
+         *
+         * @see {@link https://ts-morph.com/manipulation/formatting}
+         */
+        sourceFile.formatText();
+
         return {
           [filePath]: sourceFile.getFullText(),
         };
       }),
-
-      // Because we're returning the raw source files for TS generation we also need to separately
-      // emit out our declaration files so we can put those into a separate file in the installed
-      // SDK directory.
-      ...this.project
-        .emitToMemory({ emitOnlyDtsFiles: true })
-        .getFiles()
-        .map(sourceFile => ({
-          [path.basename(sourceFile.filePath)]: sourceFile.text,
-        })),
```

Comment:
This is no longer used and is safe to be removed. It was a remaining artifact of the previous CJS and (attempted) ESM export work where we were generating `.d.ts` files ourselves.